test/context.test.js

```diff
@@ -49,6 +49,15 @@ describe('Context', function () {
       context = new Context(event)
       expect(context.repo()).toEqual({owner: 'bkeepers-inc', repo: 'test'})
     })
+
+    it('returns error for context.repo()', function () {
+      context.payload.repository = undefined
```

Comment:
A more realistic scenario would be that `context.payload.repository` just doesn't exist (versus it being `undefined`). Its a subtle difference but it'd be better to do:

```js
delete context.payload.repository
```

Reply:
Done.